### PR TITLE
some more in fr_FR

### PR DESCRIPTION
**Pull request by kulikov-a**
Hi!
don't understand a word, but it looks like extra spaces)

### CODE REVIEW

File: fr_FR.po [hunk 1]
```diff
@@ -18490,7 +18490,7 @@ msgstr "Redirection vers le tableau de bord..."
 #. File: /usr/core/src/etc/inc/auth.inc, line: 105
 #: 
 msgid "A potential %sDNS Rebind attack%s has been detected.%sTry to access the router by IP address instead of by hostname."
-msgstr "Une attaque potentielle%sDNS Rebind% s a été détectée.%sEssayez d'accéder au routeur par adresse IP plutôt que par nom d'hôte."
+msgstr "Une attaque potentielle%sDNS Rebind%s a été détectée.%sEssayez d'accéder au routeur par adresse IP plutôt que par nom d'hôte."
```
Comment (fichtner) [inline fr_FR.po:18493]:
```suggestion
msgstr "Une attaque potentielle %sDNS Rebind%s a été détectée.%sEssayez d'accéder au routeur par adresse IP plutôt que par nom d'hôte."
```

File: fr_FR.po [hunk 2]
```diff
@@ -9329,7 +9329,7 @@ msgstr "Interface parent, adresse locale, adresse de tunnel distant, réseau de
 #. File: /usr/core/src/www/interfaces_gif.php, line: 144
 #: 
 msgid "If you are using a GIF tunnel to connect to a Hurricane Electric (he.net) Tunnel Broker on a WAN with a dynamic IP, you may want to add a %sHE.net Tunnelbroker dynamic DNS entry%s to keep your tunnel functional when your IP changes."
-msgstr "Si vous utilisez un tunnel GIF pour vous connecter à un courtier de tunnel Hurricane Electric (he.net) sur un réseau étendu avec une adresse IP dynamique, vous pouvez ajouter une entrée DNS dynamique% sHE.net Tunnelbroker%s pour que votre tunnel reste fonctionnel votre adresse IP change."
+msgstr "Si vous utilisez un tunnel GIF pour vous connecter à un courtier de tunnel Hurricane Electric (he.net) sur un réseau étendu avec une adresse IP dynamique, vous pouvez ajouter une entrée DNS dynamique %sHE.net Tunnelbroker%s pour que votre tunnel reste fonctionnel votre adresse IP change."
```
Comment (fichtner) [inline fr_FR.po:9332]:
```suggestion
msgstr "Si vous utilisez un tunnel GIF pour vous connecter à un courtier de tunnel Hurricane Electric (he.net) sur un réseau étendu avec une adresse IP dynamique, vous pouvez ajouter une %sentrée DNS dynamique HE.net Tunnelbroker%s pour que votre tunnel reste fonctionnel votre adresse IP change."
```

File: fr_FR.po [hunk 3]
```diff
@@ -20066,7 +20066,7 @@ msgstr "Souvenez-vous d'ajouter des règles de pare feu pour tous les Services P
 #. File: /usr/plugins/net/haproxy/src/opnsense/mvc/app/views/OPNsense/HAProxy/index.volt, line: 500
 #: 
 msgid "%sPublic Services:%s The HAProxy frontend. The Public Service listens for client connections, optionally applies rules and forwards client request data to the selected Backend Pool for load balancing or proxying. Every Public Service needs to be connected to a %spreviously created Backend Pool%s."
-msgstr "% sServices publics:% s L'interface HAProxy. Le service public écoute les connexions client, applique éventuellement des règles et transfère les données de demande client au pool principal sélectionné pour l'équilibrage de charge ou le proxy. Chaque fonction publique doit être connectée à un pool backend% s créé précédemment% s."
+msgstr "%sServices publics:%s L'interface HAProxy. Le service public écoute les connexions client, applique éventuellement des règles et transfère les données de demande client au pool principal sélectionné pour l'équilibrage de charge ou le proxy. Chaque fonction publique doit être connectée à un pool backend%s créé précédemment%s."
```
Comment (fichtner) [inline fr_FR.po:20069]:
```suggestion
msgstr "%sServices publics:%s L'interface HAProxy. Le service public écoute les connexions client, applique éventuellement des règles et transfère les données de demande client au pool principal sélectionné pour l'équilibrage de charge ou le proxy. Chaque fonction publique doit être connectée à un %spool backend créé précédemment%s."
```

**Comment by fichtner** [comment]
can read most of it (French in school *cough*) so I think these should be adjusted accordingly if you agree?

**Comment by kulikov-a** [comment]
hm. github did not allow to apply the suggestions from the conversation. I hope I didn't miss anything)

**Comment by fichtner** [comment]
Thanks, I'll fix the translations in the server as well after merge.

**Pull request merged**